backend/coins/btc/coin.go

```diff
@@ -130,41 +129,16 @@ func (coin *Coin) Unit() string {
 
 // FormatAmount implements coin.Coin.
 func (coin *Coin) FormatAmount(amount int64) string {
-	return strconv.FormatFloat(btcutil.Amount(amount).ToUnit(btcutil.AmountBTC), 'f',
-		-int(btcutil.AmountBTC+8), 64) + " " + coin.Unit()
-}
-
-func formatAsCurrency(amount float64) string {
-	formatted := strconv.FormatFloat(amount, 'f', 2, 64)
-	position := strings.Index(formatted, ".") - 3
-	for position > 0 {
-		formatted = formatted[:position] + "'" + formatted[position:]
-		position = position - 3
-	}
-	return formatted
+	formattedAmount := coin.FormatAmountAsJSON(amount)
```

Comment:
Maybe move JSON stuff into handlers like you did in a previous change? That looked like a good call.

Reply:
Great idea. This made me realize that FormatAmount() was completely unused, too. 

Pushed a 2nd commit.